hack/ci-e2e-kind-upgrade.sh

```diff
@@ -170,6 +170,28 @@ function set_seed_name() {
   esac
 }
 
+function run_pre_upgrade_test() {
+  case "$SHOOT_FAILURE_TOLERANCE_TYPE" in
+  node | zone)
+    make test-pre-upgrade GARDENER_PREVIOUS_RELEASE=$GARDENER_PREVIOUS_RELEASE GARDENER_NEXT_RELEASE=$GARDENER_NEXT_RELEASE
+    ;;
+  *)
+    make test-non-ha-pre-upgrade GARDENER_PREVIOUS_RELEASE=$GARDENER_PREVIOUS_RELEASE GARDENER_NEXT_RELEASE=$GARDENER_NEXT_RELEASE
+    ;;
+  esac
+}
+
+function run_post_upgrade_test() {
+  case "$SHOOT_FAILURE_TOLERANCE_TYPE" in
+  node | zone)
+    make test-post-upgrade GARDENER_PREVIOUS_RELEASE=$GARDENER_PREVIOUS_RELEASE GARDENER_NEXT_RELEASE=$GARDENER_NEXT_RELEASE
+    ;;
+  *)
+    make test-non-ha-post-upgrade GARDENER_PREVIOUS_RELEASE=$GARDENER_PREVIOUS_RELEASE GARDENER_NEXT_RELEASE=$GARDENER_NEXT_RELEASE
+    ;;
+  esac
+}
```

Comment:
The functions duplicate the make calls and the parameters. It is fairly verbose and explicit. You could also set a variable in case failure tolerance type is set and then just call something like `make test${prefix}-post-upgrade ...`.
It would be somewhat shorter in terms of lines of code and have less duplication, but may be a bit harder to read.
I have no hard opinion, which would be the better approach.

Reply:
Fixed